azurerm/resource_arm_eventhub_namespace_test.go

```diff
@@ -241,3 +382,78 @@ resource "azurerm_eventhub_namespace" "test" {
 }
 `, rInt, location, rInt)
 }
+
+func testAccAzureRMEventHubNamespace_basicWithTagsUpdate(rInt int, location string) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_eventhub_namespace" "test" {
+  name                = "acctesteventhubnamespace-%d"
+  location            = "${azurerm_resource_group.test.location}"
+  resource_group_name = "${azurerm_resource_group.test.name}"
+  sku                 = "Basic"
+  tags {
+	environment = "Production"
+  }
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMEventHubNamespace_basicWithCapacity(rInt int, location string) string {
+	return fmt.Sprintf(`
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_eventhub_namespace" "test" {
+  name                     = "acctesteventhubnamespace-%d"
+  location                 = "${azurerm_resource_group.test.location}"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  sku                      = "Basic"
+  capacity                 = 20
+}
+`, rInt, location, rInt)
+}
+
+func testAccAzureRMEventHubNamespace_basicWithCapacityUpdate(rInt int, location string) string {
```

Comment:
Refactoring suggestion. You can also combine this with `testAccAzureRMEventHubNamespace_basicWithCapacity`. You just need to introduce a parameter called `capacity`.